test/auth_cluster/test_auth_no_quorum.py

```diff
@@ -57,16 +57,6 @@ async def test_auth_no_quorum(manager: ManagerClient) -> None:
     names = [row.role for row in await manager.get_cql().run_async(f"LIST ROLES", execution_profile='whitelist')]
     assert set(names) == set(roles)
 
-
-async def trigger_snapshot(manager: ManagerClient, server: ServerInfo) -> None:
-    cql = manager.get_cql()
-    host = cql.cluster.metadata.get_host(server.ip_addr)
-    group0_id = (await cql.run_async(
-        "select value from system.scylla_local where key = 'raft_group0_id'",
-        host=host))[0].value
-    await manager.api.client.post(f"/raft/trigger_snapshot/{group0_id}", host=server.ip_addr)
```

Comment:
It's good that you extracted the function to a separate file, but you should do it in a separate commit, before introducing the snapshot test.